src/NServiceBus.Core.Tests/ApprovalFiles/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1020,14 +1027,19 @@ namespace NServiceBus.ConsistencyGuarantees
 }
 namespace NServiceBus.Container
 {
+    [System.ObsoleteAttribute("The NServiceBus dependency injection container API has been deprecated. Use the e" +
+        "xternally managed container mode to use custom containers. Will be removed in ve" +
+        "rsion 9.0.0.", true)]
     public class ContainerCustomizations
     {
-        public NServiceBus.Settings.SettingsHolder Settings { get; }
+        public ContainerCustomizations() { }
     }
+    [System.ObsoleteAttribute("The NServiceBus dependency injection container API has been deprecated. Use the e" +
+        "xternally managed container mode to use custom containers. Will be removed in ve" +
+        "rsion 9.0.0.", true)]
     public abstract class ContainerDefinition
     {
         protected ContainerDefinition() { }
-        public abstract NServiceBus.ObjectBuilder.Common.IContainer CreateContainer(NServiceBus.Settings.ReadOnlySettings settings);
```

Comment:
Wouldn't this member have to stay or doesn't it matter because it is obsoleted with error anyway?